services/crypto_get_info.proto

```diff
@@ -182,10 +182,22 @@ message CryptoGetInfoResponse {
          */
         StakingInfo staking_info = 22;
 
+        message VirtualAddressInfo {
+            /**
+             * A virtual address entry.
+             */
+            VirtualAddress virtual_address = 1;
+
+            /**
+             * The ethereum transaction nonce associated with this virtual address.
+             */
+            int64 nonce = 2;
```

Comment:
for consistency, this field should be `ethereum_nonce`

Reply:
Agree with consistency but i think we should make an exception and leave it as `nonce`